src/Tribe/Editor/Blocks/Rsvp.php

```diff
@@ -334,8 +336,7 @@ public function should_enqueue_ari() {
 	 * @return void
 	 */
 	public function rsvp_form() {
-
-		$response  = array( 'html' => '', 'view' => 'rsvp-form' );
+		$response  = [ 'html' => '', 'view' => 'rsvp-form' ];
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 9284f707455970124a753996616309d1</sub>